src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import { useMatch } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(true);
+
+  const match = useMatch('/people/:slug');
+  const selectedPersonSlug = match?.params.slug;
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 500);
```

Comment:
I don't understand why you are using here setTimeout(), instead you can do it like this (adding finally block) :
 const loadPeople = async () => {
    setIsLoading(true);

    try {
      const loadedPeople = await getPeople();

      setPeople(loadedPeople);
      setIsLoaded(true);
    } catch (error) {
      setHasError(true);
    } finally {
      setIsLoading(false);
    }
  };

  useEffect(() => {
    loadPeople();
  }, []);